### DISCUSSION TITLE
having problem installing next

### DISCUSSION THREAD

**Discussion by lexreal25** [vercel/next.js#14428] [Help]
npm init next-app nextjs-blog
Error: EPERM: operation not permitted, mkdir 'C:\Users\lex'
command not found: create-next-app

**Comment by kunalnhes** [comment]
Try running the cmd with administrator privileges.

Also If your user name contains some space say "Firstname Lastname" then run this command inside "C:\Users> DIR /X". It will give you the link name for your user. copy the link name which would be something like "FirstName~1".

Before updating verify the "npm get cache" contains any empty space or not. If yes the do this,
"npm config set cache C:\Users\[Your link name without the bracket]\AppData\Roaming\npm-cache"

**Reply by timneutkens** [reply]
> Try running the cmd with administrator privileges.

highly recommend not doing this.

**Reply by kunalnhes** [reply]
Would like to know the complications of the same. Why is it not recommended with administrative privilege?

**Reply by jamesmosier** [reply]
It opens up security issues. You are basically telling any node module you install that it now has admin rights.
